### PR TITLE
PHP 8 Support

### LINKED ISSUE #769: Tag release with PHP 8.0 support
Hey @jens1o! Could you please tag a new version so that there's a stable release with PHP 8.0 support? The Laravel community is quite progressive when it comes to upgrading. This way, we can tag a new release of [Laravel-FFMpeg](https://github.com/protonemedia/laravel-ffmpeg) as well :)

Not much has changed in the master branch since v0.16, so it can be a minor release.
https://github.com/PHP-FFMpeg/PHP-FFMpeg/compare/v0.16...master

Thanks in advance! 🙌

### PR DESCRIPTION
**Pull request by kornrunner**
| Q                  | A
| ------------------ | ---
| Bug fix?           | no
| New feature?       | yes
| BC breaks?         | no
| Deprecations?      | no
| Fixed tickets      | -
| Related issues/PRs | -
| License            | MIT

#### What's in this PR?

Support, tests and fixes for PHP 8

#### Why?

To add support for PHP 8

### CODE REVIEW

**Comment by Salman98** [comment]
if it's done then please help to merge this one , so we can use it

**Review by alexander-schranz — APPROVED** [review]
@kornrunner Thank you the changes looks good from my side. 

@jens1o I'm not able to merge this can you have a look at it, would be great if we could make a release as soon as possible with PHP 8 support. I'm not sure if there are BC Breaks in master, if there are we could go with 0.17.

File: .travis.yml [hunk 1]
```diff
@@ -30,12 +30,14 @@ matrix:
...
     - php: 7.4
+    - php: 8.0
+      dist: bionic
     - php: nightly
       env:
         - COMPOSER_DEV_STABILITY=true
 
 before_install:
-  - sudo add-apt-repository ppa:mc3man/trusty-media -y
+  - sudo add-apt-repository ppa:mc3man/${TRAVIS_DIST}-media -y
```
Comment (Romain) [inline .travis.yml:40]: @kornrunner Quick question here, will ${TRAVIS_DIST} properly match the `dist` parameter set up before?
This is for my information, I've never seen this type of variable before.

**Review by Romain — COMMENTED** [review]
Thanks for your contribution @kornrunner.
I've checked your changes, I think that they are mainly fine. I left you one comment to be sure that Travis will work properly.
I didn't have time to extensively dig into PHP8 changes, and to mention the fact that it would have been useful to setup a test version on a server using PHP8, so I'm not sure if this bundle will be fully compatible with this new version.
Hopefully Travis will help us, sort this out and, as mentionned by @alexander-schranz, we can use releases to separate versions and fix the bundle accordingly.

File: .travis.yml (hunk 1 above)
Comment (kornrunner) [reply .travis.yml:40]: Yes, that's picked up from https://docs.travis-ci.com/user/environment-variables/. You can see the replaced value in logs as well: https://travis-ci.org/github/PHP-FFMpeg/PHP-FFMpeg/jobs/748664569

**Comment by alexander-schranz** [comment]
@Romain do you see something missing for releasing this? I think it would be great to tag this.

**Comment by Romain** [comment]
@alexander-schranz I already created a tag 4 days ago, but for any reason, it has not been pushed properly.
I pushed it and created the release.
I also re-instored [SemVer](https://semver.org/) numerotation.
I don't know why a "v" character has been prepended to tags and releases at some point. I will probably drop it in the next releases.

**Comment by alexander-schranz** [comment]
@Romain oh sorry didn't see that there was a tag. Did just check the releases 🙈 . I'm fine using tags without the `v` prefix. Thx for the release 🎉 Think we can then close now the #769 issue.

**Comment by jens1o** [comment]
Sorry for the late response, I've just been suffering due to personal events related to the spread of Covid-19 and hence need to reorganize myself. Thanks Romain for hopping in!

**Comment by alexander-schranz** [comment]
@jens1o No Problem. Please take care of yourself, at the end its open source and for this who really need fixes can easily create a fork. So always take your time and take care of yourself. I wish you nice christmas holidays!

**Comment by jens1o** [comment]
Thank you @alexander-schranz. Definitely appreciated. :) Merry Christmas!

**Comment by Romain** [comment]
No worries @jens1o, we all do our best on our spare time. There are priorities... Take care.

**Pull request merged**